rituals/postmortems.md

```diff
@@ -0,0 +1,89 @@
+
+### What is a Postmortem?
+
+A postmortem is a document that examines an incident in detail, including:
+
+- An summary of what happened
+- The incident's impact
+- What caused the incident
+- How the incident was resolved
+- A detailed timeline
+- What could have prevented the incident
+
+### Why Do We Do Postmortems?
+
+The goal of the postmortem is to gain a detailed understanding of the root causes of the incident to avoid it happening again in the future. A secondary goal can be to reassure the client, since the actions taken during an incident response may not be visible to them.
+
+For postmortems to be effective at reducing repeat incidents, the review process has to incentivize teams to honestly identify root causes and fix them. For this reason, we practice **blameless postmortems** (see below).
+
+### When is a Postmortem Needed?
+
+It depends on the client or project. For applications or sites with an service-level agreement, postmortems are commonly carried out for high-severity incidents that violate the SLA. For client applications or site, a postmortem may only be called for following a major outage or quality-control problem.
+
+> "Incidents in your organization should have clear and measurable severity levels. These severity levels can be used to trigger the post-mortem process. For example, any incident Sev-1 or higher triggers the postmortem process, while the postmortem can be optional for less severe incidents." — [Atlassian](https://www.atlassian.com/blog/statuspage/incident-postmortem-writing-tips)
+
+The postmortem document should be produced within 24-48 hours of the incident's resolution, while it's still fresh in everyone's memory.
+
+> "Despite how painful an outage may have been, the worst thing you can do is to bury it and never properly close the incident in a clear and transparent way. Most humans come together in times of crisis and communication around outage post-mortems, in my experience, has always been met with positive energy, understanding comments, constructive suggestions and numerous offers to help." — [Daniel Doubrovkine says](https://artsy.github.io/blog/2014/11/19/how-to-write-great-outage-post-mortems/)
+
+### Who Completes the Postmortem?
+
+In a small company like ours, the most senior engineer with direct knowledge should be writing an outage postmortem. It's their job and responsibility to acknowledge, understand and explain what happened. For particularly sensitive topics, it may make sense to escalate this responsibility to an engineering manager, CTO, VP or Head of Engineering.
```

Comment:
Given that we don't have a CTO or Head of Engineering, you may want to re-work this paragraph.

Reply:
Good call.